src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+
+    public static void main(String[] args) throws IOException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+        chat();
     }
+
+    private static String readCommand() throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        String str = null;
+        System.out.println("Type something");
+        str = br.readLine().toLowerCase();
+        return str;
+    }
+
+    public static void greeting() {
```

Comment:
I believe method names should be in verb form; perhaps you might want to change it to greetUser instead?

Reply:
Got it.